frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.sass

```diff
@@ -117,3 +117,6 @@ app-side-menu-tabs
 .active-container
     display: flex
     align-items: center
+
+::ng-deep .menu-item-container .mat-tab-body-wrapper
```

Comment:
Maybe we can avoid using ng deep))?

Reply:
Unfortunately it's needed here, to have placeholder(appears when slot is set to be defined externally but isn't yet) cover the whole space available mat-tab-body-wrapper has to take all the space it has, which is achieved via flex-grow property